panic: assignment to entry in nil map

Hello, it's one of the greatest program.

# What did you do? (required. The issue will be **closed** when not provided.)
vuls0.5.0 put this error when I do vuls report and vuls tui.
vuls0.5.0 put the below error.

# What did you expect to happen?
vuls report -format-json
Success [scan report].

# What happened instead?

* Current Output
$ vuls report -format-json
```
[Oct 31 13:13:52]  INFO [localhost] Validating config...
[Oct 31 13:13:52]  INFO [localhost] Loaded: /home/vulsuser/results/2018-10-31T13:06:00+09:00
[Oct 31 13:13:52]  INFO [localhost] Validating db config...
[Oct 31 13:13:52]  INFO [localhost] cve-dictionary: /home/vulsuser/cve.sqlite3
[Oct 31 13:13:52]  INFO [localhost] oval-dictionary: /home/vulsuser/oval.sqlite3
[Oct 31 13:13:52]  INFO [localhost] gost: /home/vulsuser/gost.sqlite3
INFO[10-31|13:13:52] Opening DB.                              db=sqlite3
INFO[10-31|13:13:52] Migrating DB.                            db=sqlite3
[Oct 31 13:13:52]  INFO [localhost] OVAL is fresh: redhat 7.5.1804
[Oct 31 13:13:56]  INFO [localhost] yomitan: 63 CVEs are detected with OVAL
[Oct 31 13:13:56]  INFO [localhost] yomitan: 0 CVEs are detected with CPE
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/future-architect/vuls/gost.RedHat.fillFixed(0x0, 0x0, 0x1188060, 0xc42000d1c0, 0xc4204ba000, 0x3eb, 0x0)
	/home/vulsuser/go/src/github.com/future-architect/vuls/gost/redhat.go:84 +0x539
github.com/future-architect/vuls/gost.RedHat.FillWithGost(0x0, 0x0, 0x1188060, 0xc42000d1c0, 0xc4204ba000, 0xc420cb8f70, 0xc420cb8f70, 0xf8f220)
	/home/vulsuser/go/src/github.com/future-architect/vuls/gost/redhat.go:42 +0xcc
github.com/future-architect/vuls/report.FillWithGost(0x1188060, 0xc42000d1c0, 0xc4204ba000, 0xc42099b188, 0x2, 0x2)
	/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:292 +0xc9
github.com/future-architect/vuls/report.FillCveInfo(0x1187680, 0xc42000c8a0, 0x11864e0, 0xc420819b90, 0x1188060, 0xc42000d1c0, 0xc4204ba000, 0x0, 0x0, 0x0, ...)
	/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:167 +0x75d
github.com/future-architect/vuls/report.FillCveInfos(0x1187680, 0xc42000c8a0, 0x11864e0, 0xc420819b90, 0x1188060, 0xc42000d1c0, 0xc4204bbb00, 0x1, 0x1, 0xc4203daae0, ...)
	/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:86 +0x2c7
github.com/future-architect/vuls/commands.(*ReportCmd).Execute(0xc4203ec4d0, 0x1181300, 0xc420034080, 0xc4202c7e60, 0x0, 0x0, 0x0, 0x0)
	/home/vulsuser/go/src/github.com/future-architect/vuls/commands/report.go:397 +0x1491
github.com/future-architect/vuls/vendor/github.com/google/subcommands.(*Commander).Execute(0xc42009e000, 0x1181300, 0xc420034080, 0x0, 0x0, 0x0, 0x0)
	/home/vulsuser/go/src/github.com/future-architect/vuls/vendor/github.com/google/subcommands/subcommands.go:142 +0x29f
github.com/future-architect/vuls/vendor/github.com/google/subcommands.Execute(0x1181300, 0xc420034080, 0x0, 0x0, 0x0, 0xc42040c438)
	/home/vulsuser/go/src/github.com/future-architect/vuls/vendor/github.com/google/subcommands/subcommands.go:420 +0x5f
main.main()
	/home/vulsuser/go/src/github.com/future-architect/vuls/main.go:54 +0x27b
```
$ vuls tui
```
[Oct 31 12:33:47]  INFO [localhost] Validating config...
[Oct 31 12:33:47]  INFO [localhost] Loaded: /home/vulsuser/results/2018-10-31T12:12:50+09:00
INFO[10-31|12:33:47] Opening DB.                              db=sqlite3
INFO[10-31|12:33:47] Migrating DB.                            db=sqlite3
[Oct 31 12:33:47]  INFO [localhost] OVAL is fresh: redhat 7.5.1804
[Oct 31 12:33:51]  INFO [localhost] yomitan: 63 CVEs are detected with OVAL
[Oct 31 12:33:51]  INFO [localhost] yomitan: 0 CVEs are detected with CPE
panic: assignment to entry in nil map
goroutine 1 [running]:
github.com/future-architect/vuls/gost.RedHat.fillFixed(0x0, 0x0, 0x1188060, 0xc4204a0120, 0xc420496000, 0x3eb, 0x0)
/home/vulsuser/go/src/github.com/future-architect/vuls/gost/redhat.go:84 +0x539
github.com/future-architect/vuls/gost.RedHat.FillWithGost(0x0, 0x0, 0x1188060, 0xc4204a0120, 0xc420496000, 0xc420e4bce0, 0xc420e4bce0, 0xf8f220)
/home/vulsuser/go/src/github.com/future-architect/vuls/gost/redhat.go:42 +0xcc
github.com/future-architect/vuls/report.FillWithGost(0x1188060, 0xc4204a0120, 0xc420496000, 0xc42089b328, 0x2, 0x2)
/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:292 +0xc9
github.com/future-architect/vuls/report.FillCveInfo(0x1187680, 0xc420521440, 0x11864e0, 0xc4200916e0, 0x1188060, 0xc4204a0120, 0xc420496000, 0x0, 0x0, 0x0, ...)
/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:167 +0x75d
github.com/future-architect/vuls/report.FillCveInfos(0x1187680, 0xc420521440, 0x11864e0, 0xc4200916e0, 0x1188060, 0xc4204a0120, 0xc420496d80, 0x1, 0x1, 0xc4203dab40, ...)
/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:86 +0x2c7
github.com/future-architect/vuls/commands.(*TuiCmd).Execute(0xc420414000, 0x1181300, 0xc420034080, 0xc4202c7e60, 0x0, 0x0, 0x0, 0x0)
/home/vulsuser/go/src/github.com/future-architect/vuls/commands/tui.go:187 +0x81e
github.com/future-architect/vuls/vendor/github.com/google/subcommands.(*Commander).Execute(0xc42009e000, 0x1181300, 0xc420034080, 0x0, 0x0, 0x0, 0x0)
/home/vulsuser/go/src/github.com/future-architect/vuls/vendor/github.com/google/subcommands/subcommands.go:142 +0x29f
github.com/future-architect/vuls/vendor/github.com/google/subcommands.Execute(0x1181300, 0xc420034080, 0x0, 0x0, 0x0, 0xc42040c468)
/home/vulsuser/go/src/github.com/future-architect/vuls/vendor/github.com/google/subcommands/subcommands.go:420 +0x5f
main.main()
/home/vulsuser/go/src/github.com/future-architect/vuls/main.go:54 +0x27b
```

Please re-run the command using ```-debug``` and provide the output below.
vuls report -debug
```
[Oct 31 13:57:17]  INFO [localhost] Validating config...
[Oct 31 13:57:17]  INFO [localhost] Loaded: /home/vulsuser/results/2018-10-31T13:49:33+09:00
[Oct 31 13:57:17] DEBUG [localhost] ******* (centos7.5.1804): config.ServerInfo{
  ServerName:  "*******",
  User:        "*********",
  Host:        "xx.xx.xx.xx",
  Port:        "22",
  KeyPath:     "/home/vulsuser/.ssh/id_rsa",
  KeyPassword: "",
  CpeNames:    []string{},
  ScanMode:    []string{
    "deep",
  },
  DependencyCheckXMLPath: "",
  OwaspDCXMLPath:         "",
  ContainersIncluded:     []string{},
  ContainersExcluded:     []string{},
  ContainerType:          "",
  Containers:             map[string]config.ContainerSetting{},
  IgnoreCves:             []string{},
  IgnorePkgsRegexp:       []string{},
  UUIDs:                  map[string]string{},
  Memo:                   "",
  Enablerepo:             []string{},
  Optional:               map[string]interface {}{},
  Type:                   "",
  IPv4Addrs:              []string{},
  IPv6Addrs:              []string{},
  LogMsgAnsiColor:        "\x1b[32m",
  Container:              config.Container{
    ContainerID: "",
    Name:        "",
    Image:       "",
  },
  Distro: config.Distro{
    Family:  "",
    Release: "",
  },
  Mode: config.ScanMode{
    flag: 0x04,
  },
}
[Oct 31 13:57:17]  INFO [localhost] Validating db config...
[Oct 31 13:57:17] DEBUG [localhost] get cve-dictionary from sqlite3
[Oct 31 13:57:17]  INFO [localhost] cve-dictionary: /home/vulsuser/cve.sqlite3
[Oct 31 13:57:17]  INFO [localhost] oval-dictionary: /home/vulsuser/oval.sqlite3
[Oct 31 13:57:17]  INFO [localhost] gost: /home/vulsuser/gost.sqlite3
[Oct 31 13:57:17] DEBUG [localhost] open cve-dictionary db (sqlite3)
[Oct 31 13:57:17] DEBUG [localhost] Open cve-dictionary db (sqlite3): /home/vulsuser/cve.sqlite3
DBUG[10-31|13:57:17] Opening DB (sqlite3).
DBUG[10-31|13:57:17] Migrating DB (sqlite3).
[Oct 31 13:57:17] DEBUG [localhost] Open oval-dictionary db (sqlite3): /home/vulsuser/oval.sqlite3
[Oct 31 13:57:17] DEBUG [localhost] Open gost db (sqlite3): /home/vulsuser/gost.sqlite3
INFO[10-31|13:57:17] Opening DB.                              db=sqlite3
INFO[10-31|13:57:17] Migrating DB.                            db=sqlite3
[Oct 31 13:57:17] DEBUG [localhost] need to refresh
[Oct 31 13:57:17] DEBUG [localhost] Check whether oval fetched: redhat 7.5.1804
[Oct 31 13:57:17]  INFO [localhost] OVAL is fresh: redhat 7.5.1804
[Oct 31 13:57:20] DEBUG [localhost] CVE-2015-9262 is newly detected by OVAL
・・・skip・・・
[Oct 31 13:57:20] DEBUG [localhost] CVE-2018-14526 is newly detected by OVAL
[Oct 31 13:57:20]  INFO [localhost] *******: 63 CVEs are detected with OVAL
[Oct 31 13:57:20]  INFO [localhost] *******: 0 CVEs are detected with CPE
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/future-architect/vuls/gost.RedHat.fillFixed(0x0, 0x0, 0x1188060, 0xc42056f3e0, 0xc4204da000, 0x3eb, 0x0)
	/home/vulsuser/go/src/github.com/future-architect/vuls/gost/redhat.go:84 +0x539
github.com/future-architect/vuls/gost.RedHat.FillWithGost(0x0, 0x0, 0x1188060, 0xc42056f3e0, 0xc4204da000, 0xc420ca3d90, 0xc420ca3d90, 0xf8f220)
	/home/vulsuser/go/src/github.com/future-architect/vuls/gost/redhat.go:42 +0xcc
github.com/future-architect/vuls/report.FillWithGost(0x1188060, 0xc42056f3e0, 0xc4204da000, 0xc4208d1188, 0x2, 0x2)
	/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:292 +0xc9
github.com/future-architect/vuls/report.FillCveInfo(0x1187680, 0xc4202c8780, 0x11864e0, 0xc420821c20, 0x1188060, 0xc42056f3e0, 0xc4204da000, 0x0, 0x0, 0x0, ...)
	/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:167 +0x75d
github.com/future-architect/vuls/report.FillCveInfos(0x1187680, 0xc4202c8780, 0x11864e0, 0xc420821c20, 0x1188060, 0xc42056f3e0, 0xc4204dbb00, 0x1, 0x1, 0xc4203d6ae0, ...)
	/home/vulsuser/go/src/github.com/future-architect/vuls/report/report.go:86 +0x2c7
github.com/future-architect/vuls/commands.(*ReportCmd).Execute(0xc4203ea420, 0x1181300, 0xc420036080, 0xc4202c7e60, 0x0, 0x0, 0x0, 0x0)
	/home/vulsuser/go/src/github.com/future-architect/vuls/commands/report.go:397 +0x1491
github.com/future-architect/vuls/vendor/github.com/google/subcommands.(*Commander).Execute(0xc42009e000, 0x1181300, 0xc420036080, 0x0, 0x0, 0x0, 0x0)
	/home/vulsuser/go/src/github.com/future-architect/vuls/vendor/github.com/google/subcommands/subcommands.go:142 +0x29f
github.com/future-architect/vuls/vendor/github.com/google/subcommands.Execute(0x1181300, 0xc420036080, 0x0, 0x0, 0x0, 0xc42040e348)
	/home/vulsuser/go/src/github.com/future-architect/vuls/vendor/github.com/google/subcommands/subcommands.go:420 +0x5f
main.main()
	/home/vulsuser/go/src/github.com/future-architect/vuls/main.go:54 +0x27b
```

# Configuration (**MUST** fill this out):
* OS
centos7.5
* Go version (`go version`):
1.10.1
* Go environment (`go env`):
git clone from https://github.com/future-architect/vuls.git then make install.
/usr/local/go/bin/go
* Vuls environment:
v0.5.0
Hash : bc86c24

To check the commit hash of HEAD
$ vuls -v
vuls v0.5.0 bc86c24

* config.toml:
[servers.servername]
host        = "xx.xx.xx.xx"
port        = "22"
user        = "*******"
keyPath     = "/home/vulsuser/.ssh/id_rsa"
scanMode = ["deep"]

* command:
vuls report -format-json
vuls tui

--- a/gost/redhat.go
+++ b/gost/redhat.go
@@ -67,8 +67,20 @@ func (red RedHat) fillFixed(driver db.DB, r *models.ScanResult) error {
 				continue
 			}
 			cveCont := red.ConvertToModel(&redCve)
-			v, _ := r.ScannedCves[res.request.cveID]
-			v.CveContents[models.RedHatAPI] = *cveCont
+			v, ok := r.ScannedCves[res.request.cveID]
+			if ok {
+				if v.CveContents == nil {
+					v.CveContents = models.NewCveContents(*cveCont)
+				} else {
+					v.CveContents[models.RedHatAPI] = *cveCont
+				}
+			} else {
+				v = models.VulnInfo{
+					CveID:       cveCont.CveID,
+					CveContents: models.NewCveContents(*cveCont),
+					Confidences: models.Confidences{models.RedHatAPIMatch},
+				}
+			}
 			r.ScannedCves[res.request.cveID] = v
 		}
 	} else {
@@ -80,8 +92,20 @@ func (red RedHat) fillFixed(driver db.DB, r *models.ScanResult) error {
 				continue
 			}
 			cveCont := red.ConvertToModel(&redCve)
-			v, _ := r.ScannedCves[cveID]
-			v.CveContents[models.RedHatAPI] = *cveCont
+			v, ok := r.ScannedCves[cveID]
+			if ok {
+				if v.CveContents == nil {
+					v.CveContents = models.NewCveContents(*cveCont)
+				} else {
+					v.CveContents[models.RedHatAPI] = *cveCont
+				}
+			} else {
+				v = models.VulnInfo{
+					CveID:       cveCont.CveID,
+					CveContents: models.NewCveContents(*cveCont),
+					Confidences: models.Confidences{models.RedHatAPIMatch},
+				}
+			}
 			r.ScannedCves[cveID] = v
 		}
 	}
@@ -108,14 +132,10 @@ func (red RedHat) fillUnfixed(driver db.DB, r *models.ScanResult) (nCVEs int, er
 				cveCont := red.ConvertToModel(&cve)
 				v, ok := r.ScannedCves[cve.Name]
 				if ok {
-					if _, ok := v.CveContents[models.RedHatAPI]; ok {
-						v.CveContents[models.RedHatAPI] = *cveCont
+					if v.CveContents == nil {
+						v.CveContents = models.NewCveContents(*cveCont)
 					} else {
-						v = models.VulnInfo{
-							CveID:       cveCont.CveID,
-							CveContents: models.NewCveContents(*cveCont),
-							Confidences: models.Confidences{models.RedHatAPIMatch},
-						}
+						v.CveContents[models.RedHatAPI] = *cveCont
 					}
 				} else {
 					v = models.VulnInfo{
@@ -125,7 +145,6 @@ func (red RedHat) fillUnfixed(driver db.DB, r *models.ScanResult) (nCVEs int, er
 					}
 					nCVEs++
 				}
-
 				pkgStats := red.mergePackageStates(v,
 					cve.PackageState, r.Packages, r.Release)
 				if 0 < len(pkgStats) {
@@ -146,14 +165,10 @@ func (red RedHat) fillUnfixed(driver db.DB, r *models.ScanResult) (nCVEs int, er
 				cveCont := red.ConvertToModel(&cve)
 				v, ok := r.ScannedCves[cve.Name]
 				if ok {
-					if _, ok := v.CveContents[models.RedHatAPI]; ok {
-						v.CveContents[models.RedHatAPI] = *cveCont
+					if v.CveContents == nil {
+						v.CveContents = models.NewCveContents(*cveCont)
 					} else {
-						v = models.VulnInfo{
-							CveID:       cveCont.CveID,
-							CveContents: models.NewCveContents(*cveCont),
-							Confidences: models.Confidences{models.RedHatAPIMatch},
-						}
+						v.CveContents[models.RedHatAPI] = *cveCont
 					}
 				} else {
 					v = models.VulnInfo{

